src/vocgui/urls.py

```diff
@@ -56,6 +56,11 @@ def __init__(self):
 router.register(
     r"training_set/(?P<discipline_id>[0-9]+)", views.TrainingSetViewSet, "training_set"
 )
+router.register(
+    r"training_set_details/(?P<training_set_id>[0-9]+)",
+    views.TrainingSetByIdViewSet,
+    "training_set",
+)
```

Comment:
```suggestion
router.register("training_sets", views.TrainingSetByIdViewSet, "training_sets")
```
I think the viewset automatically consists of multiple views - one list view and one detail view, so we don't need to manually define the id parameter.